### ISSUE TITLE
Embedded links confused page.1.html v.s. page.1.gz.html

### ISSUE THREAD

**Issue by bhyde** [Yanpas/mandocset#3]
Awesome tool, thank you!

I made myself docset with the man pages found in an Amazon Linux 2 system.

Roughly, by doing this on an Amazon Linux 2 EC2 instance:
```  
git clone https://github.com/Yanpas/mandocset.git
sudo yum install -y python3
sudo yum install –y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
sudo yum-config-manager --enable epel
sudo yum install -y man2html
python3 mandocset.py -o Amzn2 -p /usr/share/man/ -i etc/tux.png -I etc/tux@2x.png
```
Then I copied that back to laptop, like so:

```
cd ~/.docsets
ssh $ec2_instance tar -C w/mandocset -c Amzn2.docset | tar -x
```

This docset works great, but for one flaw.

The individual pages are to found at url's like so:
    file:////Users/bhyde/.docsets/Amzn2.docset/Contents/Resources/Documents/man1/systemctl.1.gz.html

But the embedded links look like so:
    file:////Users/bhyde/.docsets/Amzn2.docset/Contents/Resources/Documents/man1/systemctl.1.html

**Comment by Yanpas** [comment]
The tool was designed for noncompressed man files, I'll fix that.
I guess this should fix (line 34):
`name = name.replace('.gz', '').replace('.bz2', '')`

**Comment by bhyde** [comment]
Added that link belows line 34...

Good news ... the links in and between the individual man pages work
Bad news ... what I think is might be called the search index still links to the compressed files which are now missing.

**Comment by bhyde** [comment]
This appears to be sufficent
```
diff --git a/mandocset.py b/mandocset.py
index b51a8e6..1412d73 100644
--- a/mandocset.py
+++ b/mandocset.py
@@ -32,6 +32,7 @@ def getPlist(name):
 
 def toHtml(inf, outdir):
 	name = os.path.basename(inf)
+	name = name.replace('.gz', '').replace('.bz2', '')
 	subp = subprocess.Popen(['man2html', inf, '-r'], stdout=subprocess.PIPE)
 	subp.stdout.readline() # skip Content-Type http header
 	with open(os.path.join(outdir, name) + '.html', 'wb') as f:
@@ -89,7 +90,7 @@ class DocsetMaker:
 							if not (new_el in self.dups):
 								toHtml(manf, outdir)
 								self.db.execute('INSERT OR IGNORE INTO searchIndex(name, type, path) VALUES (?,?,?);',
-										[name_for_db, dashtype, fname])
+										[name_for_db, dashtype, fname.replace('.gz', '').replace('.bz2', '')])
 								self.dups.add(new_el)
 							else:
 								print('\tdup skipped',fname)
```

but I really don't know what I'm doing :)

**Comment by Yanpas** [comment]
Thanks for your report and testing :)
Fixed by e36cc31
